### PR TITLE
Fix JavaScript insertion into LPSetTextOperation.h

### PR DESCRIPTION
**Pull request by jmoody**
**REBASED** Thu Jan 7 10:12 CET

### Motivation

`make webquery_headers` now inserts minified JavaScript into LPSetTextOperation.h and LPWebQuery.h

I improved the existing script:

* Fails when files are not found
* Fails when Objective-C constant is not found
* Logs what has been done
* Suppresses warnings from `calabash-js/build.sh`; use DEBUG=1 if you want to see these warnings.

```shell
$ make webquery_headers 
bundle exec bin/make/insert-js-into-webquery-headers.rb

### Minifying JavaScript ###

EXEC: /Users/moody/git/calabash/calabash-ios-server/calabash-js/build.sh
INFO: Created:
INFO:   /Users/moody/git/calabash/calabash-ios-server/calabash-js/build/calabash-min.js
INFO:   /Users/moody/git/calabash/calabash-ios-server/calabash-js/build/set_text-min.js

### LPWebQuery.h ###

INFO: Injected calabash-min.js into LP_QUERY_JS

### LPSetTextOperation.h ###

INFO: Injected set_text-min.js into LP_SET_TEXT_JS

INFO: Done!
```

### CODE REVIEW

**Comment by jmoody** [comment]
@sapieneptus I updated the scripts to use `static NSString *const`.

File: bin/make/insert-js-into-webquery-headers.rb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  def self.log_banner(string)
+    puts self.magenta("\n### #{string} ###\n")
+  end
+
+  def self.log_unix_command(string)
+    puts self.cyan("EXEC: #{string}")
+  end
+
+  def self.colorize(string, color)
+		"\033[#{color}m#{string}\033[0m"
```
Comment (sapieneptus) [inline bin/make/insert-js-into-webquery-headers.rb]: It's a shame we can't just use the `colorize` gem ...

**Comment by sapieneptus** [comment]
Looks good to me :)

File: bin/make/insert-js-into-webquery-headers.rb (hunk 1 above)
Comment (jmoody) [inline bin/make/insert-js-into-webquery-headers.rb]: Now you are just making fun of me. :)

**Comment by jmoody** [comment]
**REBASED** Thu Jan 7 10:12 CET

**Pull request merged**